lib/recaptcha/verify.rb

```diff
@@ -6,7 +6,7 @@ module Verify
     def verify_recaptcha(options = {})
       options = {:model => options} unless options.is_a? Hash
 
-      env_options = options[:env] || ENV['RAILS_ENV']
+      env_options = options[:env] || ENV['RAILS_ENV'] || Rails.env if defined? Rails.env
```

Comment:
Rails.env might be not defined ... so we need the ENV ... but we should be doing Rails.env || ENV